### PR TITLE
Per request #1130

### PR DESCRIPTION
**Pull request by codewizard1975**
timeshift: Initial inclusion into the repository

**Summary**

Resolves https://github.com/getsolus/packages/issues/1130

**Test Plan**

- Launched command line program.
- Launched UI version, prompts CRON is required and exits.

**Checklist**

- [x] Package was built and tested against unstable
- [ ] Provide patch to remove forced dependancy on cron.
- [ ] Provide patch to automatically set up systemd timer entries and activate.

### CODE REVIEW

File: packages/t/timeshift/package.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name       : Timeshift
+version    : 24.01.1
+release    : 1
+source     :
+    - https://github.com/linuxmint/timeshift/archive/refs/tags/master.mint21.tar.gz : 7272f5007de6e831e55bbae8ffe1b5bd176c304c4a86f65ef62fb6cb08b556f3
+homepage   : https://github.com/linuxmint/timeshift/tree/master.mint21
```
Comment (davidjharder) [inline packages/t/timeshift/package.yml:6]: Trim this down to `https://github.com/linuxmint/timeshift`

File: packages/t/timeshift/package.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+license    : GPL-2.0-or-later # CHECK ME
```
Comment (davidjharder) [inline packages/t/timeshift/package.yml:7]: This is the correct license, please remove the comment

File: packages/t/timeshift/package.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+component  : system.utils
+summary    : Backup and restore tool for BTRFS and RSYNC.
+description: |
+    Timeshift for Linux is an application that provides functionality similar to the System Restore feature in Windows and the Time Machine tool in Mac OS. Timeshift protects your system by taking incremental snapshots of the file system at regular intervals. These snapshots can be restored at a later date to undo all changes to the system.
+builddeps  :
```
Comment (davidjharder) [inline packages/t/timeshift/package.yml:12]: See the warning about the correct order.
Also, we prefer to use the pkgconfig() version of dependencies when they are available. For instance: `eopkg ingo libgee-devel` shows that the package provides `pkgconfig(gee-0.8)`, so you would replace `libgee-devel` with that

**Review by davidjharder — CHANGES_REQUESTED** [review]
Some initial comments;
- Nice work on this.
- Your commit message and PR description should be edited to something more like here https://help.getsol.us/docs/packaging/creating-a-new-package#commit-message-format-for-new-packages:

```
timeshift: Initial commit

Initial commit of timeshift
Resolves https://github.com/getsolus/packages/issues/1130
```

Did you get the GUI working in your testing?

File: packages/t/timeshift/package.yml (hunk 1 above)
Comment (codewizard1975) [reply packages/t/timeshift/package.yml:6]: Done, thank you.

File: packages/t/timeshift/package.yml (hunk 2 above)
Comment (codewizard1975) [reply packages/t/timeshift/package.yml:7]: Done, thank you.

File: packages/t/timeshift/package.yml (hunk 3 above)
Comment (codewizard1975) [reply packages/t/timeshift/package.yml:12]: Updated all of them.

**Comment by davidjharder** [comment]
I'm going to mark this as a draft until you've settled on a strategy for the cronie/systemd issue.

**Comment by codewizard1975** [comment]
> 
> Did you get the GUI working in your testing?

Not from the original repo.  I've forked the repo and am working on removing the hardcoded dependancy checks and will continue from there.  I'll resubmit a new PR when everything is ready and working.

**Comment by codewizard1975** [comment]
Closing due to the app needing some major work to get it to not rely on old cron system.

**Pull request closed without merging**